.travis.yml

```diff
@@ -1,42 +1,16 @@
-sudo: required
-
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - build-essential
-      - g++-4.9
-      - libcairo2-dev
-      - libpango1.0-dev
-      - libssl-dev
-      - libjpeg62-dev
-      - libgif-dev
-      - pkg-config
-
+sudo: false
 language: node_js
 matrix:
   include:
-    - env: CXX=g++-4.9
-      node_js: '6'
+    - node_js: '6'
       os: linux
-    - env: CXX=g++-4.9
-      node_js: '5'
+    - node_js: '5'
       os: linux
-    - env: CXX=g++-4.9
-      node_js: '4'
+    - node_js: '4'
       os: linux
-    - env: CXX=g++-4.9
-      node_js: '6'
+    - node_js: '6'
       os: osx
-    - env: CXX=g++-4.9
-      node_js: '5'
+    - node_js: '5'
       os: osx
-    - env: CXX=g++-4.9
-      node_js: '4'
+    - node_js: '4'
```

Comment:
Still problematic. I think we should remove the Node 4 tests for OS X, they take too long on Travis.